templates/emails/transactional/subscription-cancel.txt

```diff
@@ -12,5 +12,3 @@ Best,
 Son.
 
 -----------------------------------------
-Son NK - SimpleLogin founder.
```

Comment:
👌

Reply:
Oops. Too fast removing spaces to add the block templating. Restored.